frontend/src/components/SignIn/SignIn.js

```diff
@@ -16,41 +22,57 @@ class SignIn extends Component {
     }));
   }
 
+  handleChange = (event) => {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  handleSubmit = (event, changeAuthorizedState) => {
+    event.preventDefault();
+    loginUser(this.state)
+      .then((response) => {
+        sessionStorage.jwt = response.headers.authorization;
+        changeAuthorizedState();
```

Comment:
One thing we may want to do above this is also store the user ID, which will be in the response body. That way we can use it to make other requests later, like getting the user's data.